docs/synapse/userguides/ug011_storm_basics.rst

```diff
@@ -93,13 +64,158 @@ Working with Synapse data commonly involves three broad types of operations:
+| Remove any IP es  | n/a                                |``-#anon.vpn``                  | - Filter operation; the minus   |
+| addresses tagged  |                                    |                                |   ( ``-`` ) represents an       |
+| as anonymous VPN  |                                    |                                |   exclusion filter              |
+| infrastructure    |                                    |                                | - Hashtag ( ``#`` ) substitutes |
+|                   |                                    |                                |   for "tag"                     |
++-------------------+------------------------------------+--------------------------------+---------------------------------+ 
+| Pivot from those  |``pivot(inet:dns:a:ipv4,inet:ipv4)``|``-> inet:dns:a:ipv4``          | - Omit "from" parameter in      |
+| remaining IP es   |                                    |                                |   pivot (``inet:ipv4``) as it is|
+| addresses to any  |                                    |                                |   the primary property of our   |
+| DNS A records     |                                    |                                |   working result set (default   |
+| where those IPs   |                                    |                                |   input value)                  |
+| were present      |                                    |                                | - Arrow ( ``->`` ) substitutes  |
+|                   |                                    |                                |   for "pivot" operator          |
++-------------------+------------------------------------+--------------------------------+---------------------------------+ 
+| Pivot from those  |``pivot(inet:fqdn,inet:dns:a:fqdn)``|``inet:dns:a:fqdn -> inet:fqdn``| - Arrow ( ``->`` ) substitutes  |
+| DNS A records to  |                                    |                                |   for "pivot" operator          |
+| the domains se    |                                    |                                |                                 |
+| associated with   |                                    |                                |                                 |
+| those records     |                                    |                                |                                 |
++-------------------+------------------------------------+--------------------------------+---------------------------------+ 
+
+**Note:** Filtering (including or excluding a subset of results) is a common operation in Storm, supported by the ``filter()`` operator. Storm supports an extensive and flexible set of filtering criteria, and as a result the full operator syntax for filtering is generally complex and therefore impractical to use at the CLI. For this reason, filter operations at the CLI are generally performed using the macro syntax, as shown above.
```

Comment:
Nah, the point wasn't to "allow" people to use the filter() operator syntax, more a matter of me learning how to structure the docs around what's practical. The docs are kind of set up to continually demonstrate / contrast operator vs macro which just happens to break down here. Not a big deal.